blocks/gmo-campaign-details/gmo-campaign-details.js

```diff
@@ -0,0 +1,526 @@
+                    <div class="milestone">
+                        <span class="icon icon-trophy"></span>
+                        Milestone 3
+                    </div>
+                </div>
+                <div class="card products">
+                    <div class="card-heading h3">Products</div>
+                    <div class="product card-content">
+                        <span class="icon icon-Express"></span>
+                        Adobe Express Mobile App
+                    </div>
+                </div>
+                <div class="card audiences">
+                    <div class="card-heading h3">Audiences</div>
+                    <div class="audience card-content">
+                        <span class="icon icon-gear"></span>
+                        Existing Express Users
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div id="tab2" class="deliverables">
+            <div class="page-heading">
+                <div class="artifacts-wrapper">
+                    <span class="h3">Links to Important Artifacts</span>
+                    <div class="links">
+                        <a href="#" class="campaign-link">Creative Architecture</a>
+                        <a href="#" class="campaign-link">E2E Journeys</a>
+                        <a href="#" class="campaign-link">Marketing Brief</a>
+                    </div>
+                </div>
+                <div class="total-assets">
+                    <div class="h3">Total Assets</div>
+                    <span class="description">7</span>
+                </div>
+            </div>
+            <div class="table-wrapper">
+                <div class="table-header">
+                    <div class="header table-column column1">Content Name</div>
+                    <div class="header table-column column2">Content Type</div>
+                    <div class="header table-column column3">Channel</div>
+                    <div class="header table-column column4">Review Link</div>
+                    <div class="header table-column column5">Final Asset</div>
+                    <div class="header table-column column6">KPI</div>
+                    <div class="header table-column column7">Status Update</div>
+                    <div class="header table-column column8">Due Date</div>
+                    <div class="header table-column column9">Lead / Driver</div>
+                </div>
+                <div class="table-content">
+                </div>
+            </div>
+        </div>
+        <div id="tab3" class="calendar inactive">
+        </div>
+    </div>
+    `;
+    const tableRoot = block.querySelector('.table-content');
+    tableRoot.appendChild(rows);
+    decorateIcons(block);
+}
+function buildRows(data) {
+    console.log(data);
+    const rows = document.createElement('div');
+    const uniqueCategories = getUniqueValues(data, 'category');
+    uniqueCategories.forEach((category) => {
```

Comment:
can you add a condition to check if `category` and `subcategory` are not present then just show the deliverables without grouping. Something like this 
![image](https://github.com/hlxsites/adobe-gmo/assets/61603050/b441d1ea-d84d-42c8-a7f2-457a15b471a8)

Reply:
Yes, I'm noting a to-do right now because I'll need to think about how I want to do it.